### PR TITLE
Preparing release 1.10.0

### LINKED ISSUE #1321: Release 1.10
Tracking issue for release 1.10

- [x] https://github.com/jaegertracing/jaeger/pull/1309
- [x] https://github.com/jaegertracing/jaeger/pull/1319
- [x] go-bindata and static unification https://github.com/jaegertracing/jaeger/pull/1348
- [x] removing SASI index on dependencies table in Cassandra #1328

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves #1321

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,23 +6,37 @@ Changes by Version
 
 #### Backend Changes
 
-- Remove cassandra SASI indices [#1328](https://github.com/jaegertracing/jaeger/pull/1328)
+##### Breaking Changes
+
+- Remove cassandra SASI indices ([#1328](https://github.com/jaegertracing/jaeger/pull/1328), [@black-adder](https://github.com/black-adder))
```
Comment (pavolloffay) [inline CHANGELOG.md]: I have moved this to breaking changes even it is not. It better catches attention.

**Comment by pavolloffay** [comment]
There is only one commit https://github.com/jaegertracing/jaeger-ui/commits/master and no new version of UI has been released.

**Comment by tiffon** [comment]
@pavolloffay Per Yuri's request we cut a UI release which has two commits, which are fairly minor bug fixes.

https://github.com/jaegertracing/jaeger-ui/releases/tag/v1.0.1

**Comment by pavolloffay** [comment]
@tiffon would you like to bump it here? 

We could do patch release or wait for the next release #1343. I would like to get out 1.11 in 2 weeks.

**Pull request merged**